playbooks/ansible-test-cloud-integration-base/pre.yaml

```diff
@@ -9,11 +12,27 @@
         tox_install_siblings: false
         zuul_work_dir: "{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
 
-    - copy:
-        content: |
-            [DEFAULT]
-            vcenter_username: administrator@vsphere.local
-            vcenter_password: !234AaAa56
-            vcenter_hostname: {{ hostvars['vcenter']['nodepool']['interface_ip']}}
-            vmware_validate_certs: false
-        dest: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/test/integration/cloud-config-vcenter.ini"
+    - import_role:
```

Comment:
nit, lets add name here to give a little description of what is going on.

Reply:
Done.